lib/smack/SmackInstGenerator.cpp

```diff
@@ -834,61 +841,66 @@ void SmackInstGenerator::visitMemSetInst(llvm::MemSetInst& msi) {
 }
 
 void SmackInstGenerator::generateUnModeledCall(llvm::CallInst* ci) {
-  if (ci->getFunctionType()->getReturnType()->isVoidTy()) {
-    WARN("ignoring call to "+ci->getCalledFunction()->getName().str());
+  if (ci->getType()->isVoidTy()) {
+    //SmackWarnings::warnUnsound(ci->getCalledFunction()->getName(), currBlock, ci, true);
```

Comment:
Why is this commented out?

Reply:
This is relevant to this comment: https://github.com/smackers/smack/pull/451#discussion_r290532783. We don't want to show a lot of `llvm.dbg` intrinsics.

Reply:
Ok. But then why is is commented out, instead of completely erased?

Reply:
I fixed it.